helm/charts/nats/templates/configmap.yaml

```
@@ -99,6 +99,10 @@ data:
       {{- if .Values.nats.jetstream.uniqueTag }}
       unique_tag: {{ .Values.nats.jetstream.uniqueTag }}
       {{- end }}
+
+      {{- if .Values.nats.jetstream.max_outstanding_catchup }}
```

I will change this one into camel case like the others fyi, so would become
```suggestion
      {{- if .Values.nats.jetstream.maxOutstandingCatchup }}
```